bundles/pom.xml

```diff
@@ -5,551 +5,1083 @@
   <modelVersion>4.0.0</modelVersion>
 
   <parent>
+
```

Comment:
Please remove all of the extra spaces added to this file.

Reply:
Done. Looks like spotless updated this. I've reverted it back to the version without the spaces.